docs/content/examples/packagedata.md

```diff
@@ -0,0 +1,23 @@
+# Passing in local data into a Tesseract
+
+## Context
+You may need to include local files into your Tesseract. This guide will demonstrate how.
```

Comment:
Let's maybe extend the context here with a few more sentences (and mention artifacts, model weights, ...)